float64 tau_df
float64 tau_pf
---
bool success